tests/testthat/test-out-of-range.R

```diff
@@ -0,0 +1,16 @@
+test_that("out of range points ignored", {
+  skip_on_ci()
+  d <- tibble::tibble(id = c(1, 2),
+                      lat = c(18.2084835, 39),
+                      lon = c(-66.5858779, -85))
+
+  expect_snapshot(
```

Comment:
instead of a snapshot test, could we add other tests on `d`?  like number of rows, name of columns, and one or two values?

or you could do an expect message with a regexp string to look for?

this is only because I've been bitten before by using `skip_on_ci()` with snapshot tests and missed an actual bug.

Reply:
Yes, that's a good idea. This package probably needs more/better tests eventually too.